tests/tpot_tests.py

```diff
@@ -297,6 +297,22 @@ def test_conf_dict_3():
     assert tpot_obj.config_dict == tested_config_dict
 
 
+def test_conf_dict_4():
+    """Assert that TPOT uses seeds from custom dictionary as the starting population."""
+    tpot_obj = TPOTRegressor(config_dict='tests/test_config.py', generations=1, population_size=10)
+
+    assert isinstance(tpot_obj._pop, list)
+    assert isinstance(tpot_obj._pop[0], creator.Individual)
+
+
+def test_conf_dict_5():
+    """Assert that TPOT has a _pop attribute of the same size as the number of seeds provided."""
+    tpot_obj = TPOTRegressor(config_dict='tests/test_config.py')
+    n_seeds = len(tpot_obj._read_config_file('tests/test_config.py').population_seeds)
+
+    assert len(tpot_obj._pop) == n_seeds
```

Comment:
There seems to be just one that broke, and only in 2.7. I'll install conda for 2.7 so I can see what went wrong.